main.go

```diff
@@ -536,14 +552,26 @@ func serverListen(context *Context) error {
 		return err
 	}
 
+	// Compile the rego policy
+	allowQuery, err := rego.New(
+		rego.Query(*serverAllowQuery),
+		rego.Load([]string{*serverAllowPolicy}, nil),
+	).PrepareForEval(gocontext.Background())
```

Comment:
Looks like this block needs to be gated by the flag, so we only compile the query if both the flags for policy/query were set.

Reply:
Geez
Fixing now

Reply:
Adding integration tests while I am at it to avoid further idiocy :/